examples/darwin-framework-tool/templates/tests/ciTests.json

```
@@ -6,6 +6,7 @@
         "TestClusterComplexTypes",
         "TestEvents",
         "TestDiscovery",
+        "TestTemperatureControl",
```

```suggestion
        "Disabled due to TemperatureControl not being enabled in Matter.framework for now:",
        "TestTemperatureControl",
```
and perhaps put this down where the TestOperationalState bit is.